injected/src/features/navigator-interface.js

```diff
@@ -35,7 +37,17 @@ export default class NavigatorInterface extends ContentFeature {
                      * @throws {Error}
                      */
                     createMessageBridge(featureName) {
-                        return createPageWorldBridge(featureName, args.messageSecret);
+                        const existingBridge = store[featureName];
+                        if (existingBridge) return existingBridge;
+
+                        const bridge = createPageWorldBridge(featureName, args.messageSecret);
+
+                        if (!bridge) {
+                            throw new Error('Unable to create bridge');
```

Comment:
Looking at `createPageWorldBridge` I don't see it ever not returning, instead it just throws so I don't think will ever get called.

Reply:
yeah, just drop this bit 👍🏻